contxt/cli/main.py

```diff
@@ -23,19 +29,31 @@ def get_command(self, ctx, name):
         eval(code, ns, ns)
         return ns[name]
 
+    def __call__(self, *args, **kwargs):
+        try:
+            return self.main(*args, **kwargs)
+        except RequestException as e:
+            logger.error(f"{e} (response: {e.response.content})")
+        except Exception as e:
+            logger.error(e)
+
 
 @click.group(cls=Cli, name="contxt", context_settings=dict(help_option_names=["-h", "--help"]))
 @click.option(
     "--env",
     type=click.Choice(["production", "staging"]),
     default="production",
+    envvar="CONTXT_ENV",
     help="Environment for all API's",
 )
-@click.version_option(__version__, "-v", "--version")
+@click.option("--org", envvar="CONTXT_ORG", help="Organization slug")
+@click.option("-v", "--verbose", count=True, help="Increase verbosity")
```

Comment:
this enables `contxt -vv ...` to see every request that's being made (since it's being logged as `debug`)